test/signs/bus_test.exs

```diff
@@ -445,4 +426,17 @@ defmodule Signs.BusTest do
       :ok
     end)
   end
+
+  defp prediction(opts) do
+    %Predictions.BusPrediction{
+      direction_id: Keyword.get(opts, :direction_id),
+      departure_time: Timex.shift(Timex.now(), minutes: Keyword.get(opts, :departure)),
+      route_id: Keyword.get(opts, :route_id),
+      stop_id: Keyword.get(opts, :stop_id),
+      headsign: Keyword.get(opts, :headsign),
```

Comment:
This could be `opts[:headsign]` if you'd prefer how that reads (actually a bit more permissive in that you could use a map or whatever else).